apps/docs/components/LogoDownloadModal.js

```diff
@@ -0,0 +1,191 @@
+        borderRadius: '8px',
+        boxShadow: '0 2px 8px rgba(0, 0, 0, 0.15)',
+        maxWidth: '260px',
+        height: 'auto',
+        minHeight: '360px',
+        width: '90%',
+        zIndex: 1000,
+        boxSizing: 'border-box',
+    },
+    section: {
+        marginBottom: '16px',
+    },
+    sectionTitle: {
+        fontSize: '14px',
+        fontWeight: 'bold',
+        marginBottom: '8px',
+    },
+    buttonGroup: {
+        display: 'flex',
+        flexDirection: 'column',
+    },
+    button: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'flex-start',
+        padding: '12px 16px',
+        cursor: 'pointer',
+        backgroundColor: 'var(--ifm-background-color)',
+        color: 'var(--ifm-font-color-base)',
+        borderRadius: '4px',
+        fontSize: '14px',
+        transition: 'background-color 0.2s, color 0.2s',
+        border: 'none',
+        width: '100%',
+        marginBottom: '8px',
+        ':hover': {
+            backgroundColor: 'var(--ifm-color-primary-light)',
+            color: '#FFFFFF',
+        },
+    },
+    icon: {
+        marginRight: '8px',
+        width: '16px',
+        height: '16px',
+        fill: 'currentColor',
+    },
+});
+
+const DownloadIcon = () => (
+    <svg viewBox="0 0 24 24" {...stylex.props(styles.icon)}>
+        <path d="M19 9h-4V3H9v6H5l7 7 7-7zM5 18v2h14v-2H5z" />
+    </svg>
+);
+
+const CopyIcon = () => (
+    <svg viewBox="0 0 24 24" {...stylex.props(styles.icon)}>
+        <path d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm3 4H8c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2zm0 16H8V7h11v14z" />
+    </svg>
+);
+
+export default function LogoDownloadModal({ isOpen, onClose }) {
+    const { colorMode } = useColorMode();
+    const modalRef = useRef(null);
+
+    useEffect(() => {
+        const handleClickOutside = (event) => {
+            if (modalRef.current && !modalRef.current.contains(event.target)) {
+                onClose();
+            }
+        };
+
+        if (isOpen) {
+            document.addEventListener('mousedown', handleClickOutside);
+        }
+
+        return () => {
+            document.removeEventListener('mousedown', handleClickOutside);
+        };
+    }, [isOpen, onClose]);
+
+    if (!isOpen) return null;
+
+    const downloadFile = (url) => {
+        fetch(url)
+            .then(response => response.blob())
+            .then(blob => {
+                const link = document.createElement('a');
+                link.href = URL.createObjectURL(blob);
+                link.download = url.split('/').pop();
+                document.body.appendChild(link);
+                link.click();
+                document.body.removeChild(link);
+            });
+    };
+
+    const copyToClipboard = (text) => {
+        navigator.clipboard.writeText(text);
+    };
+
+    return (
+        <>
+            <div {...stylex.props(styles.overlay)} />
+            <div
```

Comment:
Use `<dialog>` element instead of `div` for this. Then you should also remove the `zIndex` from the styles.

Read the MDN docs on how to open the dialog element on a click.